src/indexeddb-js.js

```diff
@@ -922,17 +912,6 @@ define(['underscore'], function(_) {
     };
 
     //-------------------------------------------------------------------------
-    this.setVersion = function(version) {
```

Comment:
can you please not delete this? it is for backwards compatibility with the old indexedDB spec before `setVersion` was replaced by `onupgradeneeded`. even better, make the error message indicate that it has been deprecated in favor of onupgradeneeded :)